revscoring/datasources/meta/vectorizers.py

```diff
@@ -6,42 +6,41 @@
 """
 import os.path
 
-import fasttext
-
 from gensim.models.keyedvectors import KeyedVectors
 
 from ..datasource import Datasource
 
 ASSET_SEARCH_DIRS = ["word2vec/", "~/.word2vec/", "/var/share/word2vec/"]
-VECTOR_DIMENSIONS = 300
 
 
 class word2vec(Datasource):
     """
-    Generates vectors for a list of items generated by another
-    datasource.
+    Generates vectors using "word2vec format" for a list of items generated
+    by another datasource.
 
     :Parameters:
         words_datasource : :class:`revscoring.Datasource`
             A datasource that returns a list of words.
         vectorize_words : `function`
             a function that takes a list of words and converts it to a list
             of vectors of those words
+        keyed_vectors : `dict`
+            A list of word vectors in "word2vec format"
         name : `str`
             A name for the `revscoring.FeatureVector`
     """  # noqa
 
-    def __init__(self, words_datasource, vectorize_words, name=None):
+    def __init__(self, words_datasource, vectorize_words, keyed_vectors, name=None): 
```

Comment:
keyed_vectors should be stored in the object.  E.g. `self.keyed_vectors == keyed_vectors`

Reply:
We should also drop the "vectorize_words" argument.  We'll implement it natively.